src/main/java/ua/dymohlo/jdbcpractic/JDBCMethods.java

```diff
@@ -0,0 +1,128 @@
+package ua.dymohlo.jdbcpractic;
+
+import java.sql.*;
+
+public class JDBCMethods {
+    public static void insertIntoTableContact() {
+        try (Connection connection = ConnectionManager.getConnection()) {
+            String newMailContact = "INSERT INTO alevelgraduates.contact(contact, contact_type) VALUES ('snowSecond@gmail.com', 'mail')";
+            PreparedStatement preparedStatement = connection.prepareStatement(newMailContact);
+            preparedStatement.executeUpdate();
+            System.out.println(preparedStatement);
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void createNewPersonInTableEmployee() {
+        try (Connection connection = ConnectionManager.getConnection()) {
+            String createNewPerson = "INSERT INTO alevelgraduates.employee(first_name, lust_name, contact_id) VALUES ('Savitar','God',30)";
+            PreparedStatement preparedStatement = connection.prepareStatement(createNewPerson);
+            preparedStatement.executeUpdate();
+            System.out.println(createNewPerson);
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void selectFromTableEmployee() {
+        try (Connection connection = ConnectionManager.getConnection()) {
+            String graduates = "SELECT * from alevelgraduates.employee";
+            PreparedStatement preparedStatement = connection.prepareStatement(graduates);
+            ResultSet graduatestSet = preparedStatement.executeQuery();
+            while (graduatestSet.next()) {
+                String lustName = graduatestSet.getString("lust_name");
+                String firstName = graduatestSet.getString("first_name");
+                String contact = graduatestSet.getString("contact_id");
+                System.out.println(firstName + " " + lustName + " has contact_id # " + contact);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void deleteFromTableContact() {
+        try (Connection connection = ConnectionManager.getConnection()) {
+            int searchId = 2;
```

Comment:
Move id to method parameter and pass via method invocation.

public static void deleteFromTableContact(int id) {..}

.. deleteFromTableContact(id);